pkg/collector/corechecks/cluster/kubernetes_state.go

```diff
@@ -484,6 +488,11 @@ func isKnownMetric(name string) bool {
 	if _, found := metricNamesMapper[name]; found {
 		return true
 	}
-	_, found := metricTransformers[name]
-	return found
+	if _, found := metricTransformers[name]; found {
+		return true
+	}
+	if _, found := metricAggregators[name]; found {
+		return true
```

Comment:
Using a specific ksm metric for object count doesn't necessarily mean we submit a corresponding dd metric. I believe the definition of "known metric" is a metric that is converted into dd metric. Wdyt?

Reply:
Reading the code again where we call `isKnownMetric`, I think it doesn't have a big impact. But in any case, if we want to consider metricAggregators as known metrics, we'll need to reflect this in the function documentation.

Reply:
Yes, it seems that `isKnownMetric` is only used by a telemetry counter.
I’ve updated the description comment of the function to reflect its behaviour change.